M2/Macaulay2/packages/gfanInterface.m2

```diff
@@ -4436,6 +4436,14 @@ doc///
 	 assert equalMPL(B,Bprime)
 	 ///
 
+-- TEST gfanConvertToNewRing x NCAlgebra (cf. #3600)
+	TEST ///
+	R = QQ[a];
```

Comment:
```suggestion
	TEST ///
	debug needsPackage "gfanInterface"
	R = QQ[a];
```

Reply:
@felixlotter you should be able to commit and add this change directly here.

@d-torrance I think for small fixes giving suggestions this way would help contributors quickly add the changes to fix the failing tests.

Reply:
Thank you!